numpy/core/tests/test_regression.py

```diff
@@ -2411,6 +2411,79 @@ def test_field_access_by_title(self):
             if HAS_REFCOUNT:
                 assert_(base <= sys.getrefcount(s))
 
+    @pytest.mark.slow
+    @pytest.mark.skipif(not sys.platform.startswith('linux'),
+                        reason="linux only")
+    @pytest.mark.skipif(sys.version_info < (3,4), reason="lacks venv")
+    def test_pickle_across_versions(self, tmpdir):
+        # gh-12837
+        from subprocess import Popen, PIPE
+        import os
+        # Create a virtual environment, pip install 1.13 into it
+        new_python = str(tmpdir / 'bin' / 'python')
+        stdout = []
+        stderr = []
+        for cmd in [(sys.executable, '-mvenv', '--copies', str(tmpdir)),
+                    (new_python, '-mpip', 'install', 'numpy==1.13.0'),
+                   ]:
+            p = Popen(cmd, stdout=PIPE, stderr=PIPE)
+            o, e = p.communicate()
+            res = p.wait()
+            stdout.append(o.decode('utf8'))
+            stderr.append(e.decode('utf8'))
+            if res != 0:
+                print('stdout:\n', '    \n'.join(stdout))
+                print('stderr:\n', '    \n'.join(stderr))
+                assert res == 0
+        # Create a test pickle file
+        testfile = tmpdir / 'test.pkl'
+        a = np.ones((10, 10), dtype='int32')
+        with open(testfile, 'wb') as fid:
+            pickle.dump(a, fid)
+            pickle.dump(np.uint64(10), fid)
+        # Use \n to allow pretty formatting here but no SyntaxError in Popen
+        # due to lines beginning with spaces
+        cmd = [new_python, '-s', '-c', f'''
+            \n# -------Make sure _multiarray_umath cannot be imported --------
```

Comment:
I don't think you need the `\n` here, line breaks are preserved with the triple quotes.

Reply:
If I don't use `\n` I need to left-justify the text in the file, so the block would not be nicely indented. Would that be OK?

Reply:
used dedent